stable_baselines3/common/callbacks.py

```diff
@@ -435,3 +435,47 @@ def _on_step(self) -> bool:
             self.last_time_trigger = self.num_timesteps
             return self._on_event()
         return True
+
+
+class StopTrainingOnMaxEpisodes(BaseCallback):
+    """
+    Stop the training once a maximum number of episodes are played.
+
+    For multiple environments presumes that, the desired behavior is that the agent trains on each env for ``max_episodes``
+    and in total for ``max_episodes * n_envs`` episodes.
+
+    :param max_episodes: (int) Maximum number of episodes to stop training.
+    :param verbose: (int)
```

Comment:
Add description for `verbose=1` (prints out when training was killed).

Reply:
Aiming at code consistency followed other examples that did not add a description for verbose parameters.

Reply:
@araffin What do you think?

Reply:
examples are not always perfect... and more doc is always welcomed ;)